frameworks/Java/redkale/redkale-graalvm.dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM maven:3.8.1-openjdk-17-slim as maven
+FROM maven:3.6.3-openjdk-16-slim as maven
 WORKDIR /redkale
 COPY src src
 COPY conf conf
 COPY pom.xml pom.xml
 RUN mvn package -q
 
-FROM openjdk:17-jdk-slim
+FROM ghcr.io/graalvm/graalvm-ce:latest
```

Comment:
Can you change this from `:latest` to a specific version, please

Reply:
ok,  i change 21.1.0